_topic_map.yml

```diff
@@ -1094,6 +1094,12 @@ Topics:
     File: odo-cli-reference
   - Name: odo release notes
     File: odo-release-notes
+- Name: Helm
+  Dir: helm
+  Distros: openshift-enterprise
+  Topics:
+  - Name: Getting started with Helm on OpenShift
```

Comment:
Probably should be "{product-title}" instead of OpenShift

Reply:
I've replaced with the attribute but now it doesn't render properly in the preview. Should we use attributes in the topic_map file?

Reply:
You are correct, sorry! I didn't notice that this was in the topic map, attributes don't replace properly in the topic map. I'd suggest then saying "OpenShift Container Platform" instead of just "OpenShift". 

And also be sure to replace the title of the assembly to match.